02week/pigLatin.js

```diff
@@ -10,7 +10,51 @@ const rl = readline.createInterface({
 
 function pigLatin(word) {
 
+  word = word.toLowerCase();
```

Comment:
we want to try and avoid mutating data, so store the formatted word into a new const and use that